Doc/library/curses.rst

```diff
@@ -562,13 +572,13 @@ The module :mod:`curses` defines the following functions:
 .. function:: tigetstr(capname)
 
    Return the value of the string capability corresponding to the terminfo
-   capability name *capname*.  ``None`` is returned if *capname* is not a string
+   capability name *capname* as a byte string.  ``None`` is returned if *capname* is not a string
    capability, or is canceled or absent from the terminal description.
 
 
 .. function:: tparm(str[, ...])
 
-   Instantiate the string *str* with the supplied parameters, where *str* should
+   Instantiate the byte string *str* with the supplied parameters, where *str* should
```

Comment:
I afraid this will cause an ambiguity between capability name and value. The first argument of `tparm()` is not a capability name. It is usually a result of `tigetstr()`.